sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/gitlab.yaml

```diff
@@ -0,0 +1,34 @@
+provider:
```

Comment:
Nice-to-have: Add links to GitLab documentation on how to rotate and/or revoke tokens.